src/components/NewRuntimeModal.js

```diff
@@ -339,6 +340,7 @@ export const NewRuntimeModal = withModalDrawer({ width: 675 })(class NewRuntimeM
             ...(jupyterUserScriptUri && { jupyterUserScriptUri }),
             ...(cloudService === cloudServices.GCE ? {
               machineType: masterMachineType || defaultGceMachineType,
+              bootDiskSize: existingRuntime?.bootDiskSize,
```

Comment:
should we specify an optional value here to act as a stub in cost calculated if an existing runtime doesn't exist? I think we discussed wanting something like `existingRuntime?.bootDiskSize || 50`

Reply:
That'd be safer 👍🏼 

Let's not use a magic number like `50` though. Could use `DEFAULT_DISK_SIZE` which equals `50` or define a `DEFAULT_GCE_BOOT_DISK_SIZE` or something if we want to default to something other than `60` after increasing the GCE boot disk size.

Reply:
@jdcanas , when we were talking about it, we thought we need to have a default value becuz we were thinking cost is displayed via this code if there's no existing runtime, but I don't think that's the case (I tested UI without any runtime, and things look correct)....`existingRuntime?` is already checking if `existingRuntime` is null I think, and if there's an existing runtime, `bootDiskSize` will always exist for gce runtimes.

Reply:
its safe in terms of not accessing the value on an `undefined` variable. So we are safely handling the case where existingRuntime doesn't exist. However, in this case `existingRuntime.bootDiskSize` will be undefined. I think we do want to include a default value to be used in the cost calculation. See attached screenshot (I added these prints at beginning of `getPendingRuntimeConfig`)
![Screen Shot 2021-04-21 at 9 11 02 AM](https://user-images.githubusercontent.com/6465084/115559515-c4baf600-a281-11eb-9042-044988544ed2.png)

Reply:
`bootDiskSize` is a required field in leo DB for GCE. Did you mock out leo's response or something?

Reply:
talked with @jdcanas on zoom...there's default added in src/libs/runtime-utils.js now.